jamiehs/bootstrap-sass-middleman
# Bethany/Upgrade to Middleman 4

IN PROGRESS 

Also removed Compass, added Autoprefixer instead. 
### To Test
- Run `bundle install` to install updated Gems
- Start `middleman`, open browser, and edit a CSS file and index.html file to confirm that LiveReload is working
- Run `middleman build` to make sure there are no errors
#### Possible bugs/downsides

Livereload reloads all of a file 'type' when one file is edited. For example, editing test.html will also reload all tracked *html files. New bug with Middleman 4 and LR? Not sure. Limited support for LR
